readme.md

```diff
@@ -2,6 +2,7 @@
 
 [![Join the chat at https://gitter.im/rebornix/vscode-ruby](https://badges.gitter.im/rebornix/vscode-ruby.svg)](https://gitter.im/rebornix/vscode-ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![Build Status](https://travis-ci.org/rubyide/vscode-ruby.svg?branch=master)](https://travis-ci.org/rubyide/vscode-ruby) [![Build status](https://ci.appveyor.com/api/projects/status/s4sv1fwpjeqmgnhd?svg=true)](https://ci.appveyor.com/project/rebornix/vscode-ruby)
 
+#**This extension is now actively developed under [rubyide](https://github.com/rubyide/vscode-ruby)**
 
```

Comment:
I suppose you are still fetching commits from my repo, let's switch to `rubyide/vscode-ruby`.